pkg/server/home_workspaces.go

```diff
@@ -175,8 +185,15 @@ func buildLocalInformersAccess(kubeSharedInformerFactory kubernetesinformers.Sha
 		getClusterRoleBinding: func(workspace logicalcluster.Name, name string) (*rbacv1.ClusterRoleBinding, error) {
 			return crbLister.Get(clusters.ToClusterAwareKey(workspace, name))
 		},
-		getAPIBinding: func(clusterName logicalcluster.Name, name string) (*apisv1alpha1.APIBinding, error) {
-			return apiBindingLister.Get(clusters.ToClusterAwareKey(clusterName, name))
+		getTenancyAPIBinding: func(clusterName logicalcluster.Name) (*apisv1alpha1.APIBinding, error) {
```

Comment:
how about a `*APIBinding, bool, error` return value? I don't like to mix errors and expected values.

Reply:
ok